.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: c
+#env:
+#  global:
+#    - GAPROOT=gaproot
+#    - COVDIR=coverage
+#    # list of package to compile; io and profiling are always needed for coverage reports
+#    - GAP_PKGS_TO_BUILD="io profiling"
+
+addons:
+  apt_packages:
+    - libgmp-dev
+    - libreadline-dev
+    - zlib1g-dev
+
+#matrix:
+#  include:
+#    - env: CFLAGS="-O2"
+
+branches:
+  only:
+    - master
+
+before_script:
+  - DONT_FETCH_GAP_JULIA_PKG=yes etc/install_julia_gap.sh
+
+script:
+  - cd JuliaInterface
+  - ./configure $TRAVIS_BUILD_DIR/gap && make && $TRAVIS_BUILD_DIR/gap/gap tst/testall.g
+
+#after_script:
+#  - bash scripts/gather-coverage.sh
+#  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Obviously this file still needs some cleanup, but I first want to get it working, and perhaps also add codecov support